conf/__init__.py

```diff
@@ -61,6 +61,12 @@
 bittrex_api_key = os.getenv("BITTREX_API_KEY")
 bittrex_secret_key = os.getenv("BITTREX_SECRET_KEY")
 
+# Bitfinex
+# this values ​​set by empirically way, because the bitfinex-market does not have
+# these values. maybe later it will be in market-api.
+bitfinex_base_increment = 1e-8
+bitfinex_quote_increment = 0.01
+
```

Comment:
Since this is only used in `BitfinexAPIDataSource` you can just have it in there, it is best to not have it in `conf/__init__.py`.

Reply:
hmm, github not marked this code, maybe because I used rebase.  
[You can find changes in here ](https://github.com/CoinAlpha/hummingbot/pull/1190/commits/3a348e1600bdd303d184e41c12d7b0283579630b)